holmes/config.py

```
@@ -205,6 +148,32 @@ def model_post_init(self, __context: Any) -> None:
+        """
+        Load configuration from file and merge with CLI options.
+
+        Args:
+            config_file: Path to configuration file
+            **kwargs: CLI options to override config file values
+
+        Returns:
+            Config instance with merged settings
+        """
+        if config_file is not None and config_file.exists():
+            logging.debug(f"Loading config from {config_file}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)